tasks/collections.go

```diff
@@ -1,31 +1,48 @@
 package tasks
 
+import "fmt"
+
 // Task 1: Write a function that creates and prints a slice of integers with values 1 to 5.
 
 func CreateAndPrintSlice() {
 	// Implement the code to create and print a slice of integers.
+	demo := []int{5, 7}
```

Comment:
Read the instructions carefully, task is to print a slice of integers with values 1 to 5.